mqtt/mqtt-bridge/src/settings.rs

```diff
@@ -31,7 +33,16 @@ impl BridgeSettings {
         let mut config = Config::new();
 
         config.merge(File::from_str(DEFAULTS, FileFormat::Json))?;
-        config.merge(Environment::new())?;
+        config.merge(Environment::new().separator(ENV_VAR_SEPARATOR))?;
+
+        if let Ok(default_storage_memory) =
+            config.get_bool(&DEFAULT_STORAGE_MEMORY_VAR.to_ascii_lowercase())
+        {
+            if default_storage_memory {
+                config.set("storage.type", "memory")?;
+                config.set("storage.max_size", 1024 * 1024 * 1024)?;
+            }
+        }
```

Comment:
I have added an env var to default over to memory storage, let me know if there is a better way or name that you can think of here... just went with something simple